README.md

```diff
@@ -1,15 +1,15 @@
-# ![Blendid](https://raw.githubusercontent.com/vigetlabs/blendid/master/extras/blendid-logo.png)
+# ![Blendid](https://raw.githubusercontent.com/vigetlabs/blendid/master/root/blendid-logo.png)
 
 [![Build Status](https://travis-ci.org/vigetlabs/blendid.svg?branch=static-server)](https://travis-ci.org/vigetlabs/blendid)
 [![Stories in Ready](https://badge.waffle.io/vigetlabs/blendid.png?label=ready&title=Ready)](https://waffle.io/vigetlabs/blendid)
 
-**Blendid** *(formerly known as Gulp Starter)* is a delicious stand-alone blend of tasks and build tools poured into [Gulp](http://gulpjs.com/) to form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
+**Blendid** is a delicious stand-alone blend of tasks and build tools that form a full-featured modern asset pipeline. It can be used as-is as a static site builder, or can be configured and integrated into your own development environment and site or app structure.
 
 ## Quick start on a fresh project (empty directory)
 ```bash
 yarn init
 yarn add blendid
-yarn run blendid -- init
+yarn run blendid init
 yarn run blendid
```

Comment:
I think this _could_ just be `yarn blendid init`

Reply:
i think you are correct. not sure why the `run` is in there in the first place, so simplifying it may be better for everyone